database/migrations/2024_03_18_164714_add_note_to_checkout_acceptance_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddDeclineMsgToCheckoutAcceptanceTable extends Migration
```

Comment:
One more change:
```suggestion
class AddNoteToCheckoutAcceptanceTable extends Migration
```

I'm getting the following exception when I try to migrate it now:
```
└─[$]› art migrate

   Symfony\Component\ErrorHandler\Error\FatalError

  Cannot declare class AddDeclineMsgToCheckoutAcceptanceTable, because the name is already in use

  at database/migrations/2024_03_18_164714_add_note_to_checkout_acceptance_table.php:7
      3▕ use Illuminate\Database\Migrations\Migration;
      4▕ use Illuminate\Database\Schema\Blueprint;
      5▕ use Illuminate\Support\Facades\Schema;
      6▕
  ➜   7▕ class AddDeclineMsgToCheckoutAcceptanceTable extends Migration
      8▕ {
      9▕     /**
     10▕      * Run the migrations.
     11▕      *
     ```